src/utils/index.ts

```diff
@@ -35,5 +35,5 @@ export function byggPamUrl(fnr: string, path = '/cv') {
 
 export function lagPersonforvalterLenke(aktoerIdEllerFnr: string) {
     //Personforvalteren skal takle både aktørid og fnr.
```

Comment:
Denne kommentaren sto der fra før, men jeg får ikke treff i pdl-web dersom jeg legger på ?fnr= og putter inn fnr, eller hvis jeg putter fnr i stedet for aktørid så jeg lurer litt på om dette stemmer og hvis den skal støtte fnr hvordan.. jeg klarer ikke finne det ut via loggene heller.